wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+    <img src="../../../assets/img/pexels-dids-2508565.jpg">
+    <div class="title">
+        Wist Scoreboard Generator
+        <div class="description">
+            Genereaza tabele de scor pentru jocul de carti Wist. 
+            Tot ce trebuie sa faci este sa introduci jucatorii si optiunile de joc.
```

Comment:
You shouldn't build applications in Romanian. Please use https://angular.io/guide/i18n-overview and develop the application English first, but with support for Romanian